charts/rasa-x/values.yaml

```diff
@@ -183,10 +187,20 @@ rasa:
   jaegerSidecar: "false"
   livenessProbe:
     enabled: true
+    # liveness probe path
+    path: "/"
     # initialProbeDelay for the `livenessProbe`
     initialProbeDelay: 10
     # scheme to be used by the `livenessProbe`
     scheme: "HTTP"
+  readinessProbe:
+    enabled: true
+    # readiness probe path
+    path: "/"
```

Comment:
As per the [ticket](https://rasahq.atlassian.net/jira/software/c/projects/INFRA/boards/38?modal=detail&selectedIssue=INFRA-403), should the default value for this not be `/status`?

Reply:
@amn41 What would you expect here as the default value?